lib/profile/index.js

```diff
@@ -51,6 +51,7 @@ function defaults ( ) {
     , maxUAMSMBBasalMinutes: 30 // maximum minutes of basal that can be delivered as a single SMB when IOB exceeds COB
     , SMBInterval: 3 // minimum interval between SMBs, in minutes.
     , bolus_increment: 0.1 // minimum bolus that can be delivered as an SMB
+    , maxDelta_bg_threshold: 0.2 // maximum % change in bg to use SMB, above that will disable SMB
```

Comment:
This is technically a multiple, not a percent.

Reply:
Agree, have removed the %.